python/mathsys/bin/web/compile.sh

```
@@ -1,55 +1,44 @@
+;;
+
+;; HEAD -> MODULE
+(module
+
+;; HEAD -> IMPORTS
+(import "env" "memory" (memory 0))
+(import "sys" "call1" (func $call1 (param i32 i32)))
+(import "sys" "call60" (func $call60 (param i32)))
+
+EOF
+    cat >> all.wat << 'EOF'
```

Shellcheck (0.10.0)

[style] Consider using { cmd1; cmd2; } >> file instead of individual redirects.

(SC2129)